### PR TITLE
Update automatic-vm-guest-patching.md to indicate availability timelines for Windows Server 2025 Azure Edition

### PR DESCRIPTION
**Pull request by kjohn-msft**
The lists have been updated to prioritize clarity on the Hotpatchable images, and a message added on timelines for newer images.

### CODE REVIEW

**Comment by maulikshah23** [comment]
#sign-off

**Comment by maulikshah23** [comment]
#approved

**Review by maulikshah23 — APPROVED** [review]

**Comment by Court72** [comment]
@kjohn-msft, @maulikshah23,

The statement "Support for the following Windows Server 2025 images is currently being rolled out across the world with coverage expected everywhere by mid-December 2024" constitutes a future promise. Documentation should only describe what is possible with the released product. Can you revise the content to remove that statement?

For more information, see [What can I publish on Microsoft Learn?](https://review.learn.microsoft.com/help/platform/contribute-get-started-channel-guidance?branch=main#unauthorized-content)

After you make these changes, `#sign-off` again to start another review.

Thanks
#hold-off

**Comment by Court72** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by maulikshah23** [comment]
@sign-off

**Comment by maulikshah23** [comment]
@Court72 - please see the updated text, please review and merge immediately.

**Comment by Court72** [comment]
This PR is ready for merge. Due to the upcoming Ignite conference, all merging and publishing is on hold until the morning of the keynote address.

The PR review team will resume merging on the morning of the keynote address for the regularly scheduled daily publish cycles. You don't need to take further action on this PR; we'll merge it on the day of the keynote address.

**Don't add more commits to this reviewed PR.** If you need to make additional changes to these files, email techdocprs for guidance about how to proceed.

#label:"missed-content-complete","event-triaged-to-10AM"

File: articles/virtual-machines/automatic-vm-guest-patching.md [hunk 1]
```diff
@@ -79,29 +79,30 @@ As a new rollout is triggered every month, a VM will receive at least one patch
...
+
+| Publisher               | OS Offer      |  Sku               |
+|-------------------------|---------------|--------------------|
+| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-core |
+| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-core-smalldisk |
+| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-hotpatch |
+| MicrosoftWindowsServer  | WindowsServer | 2022-datacenter-azure-edition-hotpatch-smalldisk |
+
+> [!IMPORTANT] 
+> Support for the following **Windows Server 2025** images is currently being rolled out. Regions where support doesn't exist yet may return a HTTP 400 client error indicating an unsupported status in the interim. You may successfully use one of the regions with support like UK South, East Asia or West Central US instead. This advisory will be removed when support is available everywhere.
```
Comment (ju-shim) [inline articles/virtual-machines/automatic-vm-guest-patching.md:92]:
```suggestion
> Not all regions currently support **Windows Server 2025** images. Regions where support doesn't exist yet may return a HTTP 400 client error, indicating an unsupported status. Instead, use another region that does provide support, such as, UK South, East Asia, or West Central US.
```

**Review by ju-shim — CHANGES_REQUESTED** [review]

File: articles/virtual-machines/automatic-vm-guest-patching.md (hunk 1 above)
Comment (kjohn-msft) [reply articles/virtual-machines/automatic-vm-guest-patching.md:92]: @ju-shim We can take that change but will need to be more specific then --
Not all regions currently support **Windows Server 2025 Azure Edition** images. Regions where support doesn't exist yet may return a HTTP 400 client error, indicating an unsupported status. Instead, use another region that does provide support, such as, UK South, East Asia, or West Central US.

Comment (ju-shim) [reply articles/virtual-machines/automatic-vm-guest-patching.md:92]: @kjohn-msft - Yeah feel free to make that change, it's perfect. Let me know when you are done, and I can sign this off to go live on Nov 19th at 10am (PST). Per my email thread response, we are in a companywide publishing freeze for Ignite, so all content signed off right now will go love that morning and can't go sooner. Thank you! :)

Comment (ju-shim) [reply articles/virtual-machines/automatic-vm-guest-patching.md:92]: I updated my suggested so it's easy to add as a commit.

**Pull request merged**